decentraland/unity-explorer
# fix: Agressively reduce avatar texture pre-allocation

## What does this PR change?

Reduce avatar allocation to be able to instantiate aprox 25 avatars (assuming 10 wearables per avatar)

BEFORE

<img width="1728" alt="Screen Shot 2024-09-16 at 12 25 33" src="https://github.com/user-attachments/assets/784e1047-a94c-46ee-8980-7a8908b324b2">

AFTER 

<img width="1728" alt="Screen Shot 2024-09-16 at 13 51 43" src="https://github.com/user-attachments/assets/a9762d44-90ef-4125-88bc-e2bb76cff93c">



## How to test the changes?



1. Launch the explorer
2. Instantiate avatars using the debug tool and check everything is okay

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
